documentation/api/notation.rst

```diff
@@ -162,6 +162,67 @@ For version 1, 2 and 3 of the API, only equidistant timeseries data is expected
 - "duration" should also be a multiple of the sensor resolution.
 
 
+.. _describing_flexibility:
+
+Describing flexibility
+^^^^^^^^^^^^^^^^^^^^^^^
+
+To compute a schedule, FlexMeasures needs to know state and context information which goes beyond
+the usual time series of the asset.
+
+We've grouped these two sources of information in two:
+
+Flex model
+""""""""""""
+
+The flexibility model describes to the scheduler what the flexible asset's state is,
+and what constraints or preferences should be taken into account.
+
+Which specific flexibility model is relevant in your scheduler usually relates to the sensor's asset type.
```

Comment:
- ~specific flexibility model~ -> type of flexibility model? If so, move these two lines to after the sentences about inferring missing values (replacing the empty line - currently, 184) with these two lines). We then end up with first a paragraph about flexibility models, and then a paragraph about types of flexibility models, which should improve readability.
- ~relevant in your scheduler~ -> relevant to a scheduler